### PR TITLE
add word-search stub to fix build

### PR DESCRIPTION
**Pull request by msomji**
Related to #298. Added stub for this exercise to fix build . More work remains for other exercises.

### CODE REVIEW

File: exercises/word-search/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@types/jest": "^24.0.18",
     "@types/node": "^12.7.12",
-    "@typescript-eslint/eslint-plugin": "^2.5.0",
+    "@typescript-eslint/eslint-plugin": "^2.3.3",
```
Comment (SleeplessByte) [inline exercises/word-search/package.json]: Why this?

Comment (msomji) [reply exercises/word-search/package.json]: The make command complained that the word-search package.json was not matching the package.json that is in the root/common dir. 2.3.3 is what was in the common package.json.

Comment (SleeplessByte) [reply exercises/word-search/package.json]: I'm more wondering why this was not 2.3.3 in the first place :/.

File: exercises/word-search/word-search.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default class WordSearch {
+  public find(words: string[]) {
+    throw new Error('Implement this function')
+  }
+}
```
Comment (SleeplessByte) [inline exercises/word-search/word-search.ts]: Can you end on an empty new line?

File: exercises/word-search/package.json (hunk 1 above)
Comment (SleeplessByte) [reply exercises/word-search/package.json]: Ah I figured it out. The CI was broken when this exercise was added, so we didn't actually check this. Thanks for fixing!

**Pull request merged**